boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dtsi

```diff
@@ -151,43 +151,43 @@
 	vbat_sensor: ina231@44 {
 		compatible = "ti,ina230";
 		reg = <0x44>;
-		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
-					INA230_CONV_TIME_4156,
-					INA230_CONV_TIME_4156,
-					INA230_AVG_MODE_1024)>;
+		adc-mode = "Bus and shunt voltage continuous";
+		vbus-conversion-time-us = <4156>;
+		vshunt-conversion-time-us = <4156>;
+		avg-count = <1024>;
 		current-lsb-microamps = <1>;
 		rshunt-micro-ohms = <510000>;
 	};
 
 	vdd1_codec_sensor: ina231@45 {
 		compatible = "ti,ina230";
 		reg = <0x45>;
-		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
-					INA230_CONV_TIME_4156,
-					INA230_CONV_TIME_4156,
-					INA230_AVG_MODE_1024)>;
+		adc-mode = "Bus and shunt voltage continuous";
+		vbus-conversion-time-us = <4156>;
+		vshunt-conversion-time-us = <4156>;
+		avg-count = <1024>;
 		current-lsb-microamps = <1>;
 		rshunt-micro-ohms = <2200000>;
 	};
 
 	vdd2_codec_sensor: ina231@41 {
 		compatible = "ti,ina230";
 		reg = <0x41>;
-		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
-					INA230_CONV_TIME_4156,
-					INA230_CONV_TIME_4156,
-					INA230_AVG_MODE_1024)>;
+		adc-mode = "Bus and shunt voltage continuous";
+		vbus-conversion-time-us = <4156>;
+		vshunt-conversion-time-us = <4156>;
+		avg-count = <1024>;
 		current-lsb-microamps = <1>;
 		rshunt-micro-ohms = <2200000>;
 	};
 
 	vdd2_nrf_sensor: ina231@40 {
 		compatible = "ti,ina230";
 		reg = <0x40>;
-		config = <INA230_CONFIG(INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT,
-					INA230_CONV_TIME_4156,
-					INA230_CONV_TIME_4156,
-					INA230_AVG_MODE_1024)>;
+		adc-mode = "Bus and shunt voltage continuous";
```

Comment:
just seen this property, this looks really weird. Is devicetree being reviewed properly these days?

Reply:
Had the same thought to be honest, I'd suggest reviewing it.

Reply:
It is a DT enum for the modes from the datasheet.  Originally I had done C macros like the INA230 style here for the IN237, but in the review it was mentioned that this was the "Device Tree way" to do things with the enum (see https://github.com/zephyrproject-rtos/zephyr/pull/60717#discussion_r1275458818).

If you are just concerned about the string, I could restore the "INA230_OPER_MODE_SHUNT_BUS_VOLTAGE_CONT" macro (should probably be a C enum actually) and add those to the INA237 as well.

Reply:
This should likely be a combination of boolean options, I've never seen string properties describing configuration.

Reply:
I think enum is fine here since it maps well with the datasheet but the format looks off, a bit too descriptive. Maybe we should have a guideline with few examples for binding enum format. @galak @mbolivar-ampere